src/BloomExe/Book/BookStarter.cs

```diff
@@ -419,14 +417,10 @@ public static void SetupIdAndLineage(XmlElement parentPageDiv, XmlElement childP
 			}
 		}
 
-
-
-
-
 		private string GetInitialName(string sourcePath, string parentCollectionPath)
 		{
 			var storage = _bookStorageFactory(sourcePath);
-			var name = storage.Dom.GetMetaValue("defaultNameForDerivedBooks",  Path.GetFileName(sourcePath));
+			var name = storage.Dom.GetMetaValue("defaultNameForDerivedBooks",  "Book");
```

Comment:
But above, comments say defaultNameForDerivedBooks no longer has any use at all, and code removes it.